src/Data/Bool/Properties.agda

```diff
@@ -21,14 +25,21 @@ open import Induction.WellFounded using (WellFounded; Acc; acc)
 open import Level using (Level; 0ℓ)
 open import Relation.Binary.Core using (_⇒_)
 open import Relation.Binary.Structures
-  using (IsPreorder; IsPartialOrder; IsTotalOrder; IsDecTotalOrder; IsStrictPartialOrder; IsStrictTotalOrder)
+  using (IsPreorder; IsPartialOrder; IsTotalOrder; IsDecTotalOrder
+  ; IsStrictPartialOrder; IsStrictTotalOrder)
 open import Relation.Binary.Bundles
-  using (Setoid; DecSetoid; Poset; Preorder; TotalOrder; DecTotalOrder; StrictPartialOrder; StrictTotalOrder)
+  using (Setoid; DecSetoid; Poset; Preorder; TotalOrder; DecTotalOrder;
+  StrictPartialOrder; StrictTotalOrder)
 open import Relation.Binary.Definitions
-  using (Decidable; DecidableEquality; Reflexive; Transitive; Antisymmetric; Minimum; Maximum; Total; Irrelevant; Irreflexive; Asymmetric; Trans; Trichotomous; tri≈; tri<; tri>; _Respects₂_)
-open import Relation.Binary.PropositionalEquality.Core
-open import Relation.Binary.PropositionalEquality.Properties
-open import Relation.Nullary.Decidable.Core using (True; yes; no; fromWitness)
+  using (Decidable; DecidableEquality; Reflexive; Transitive; Antisymmetric;
+  Minimum; Maximum; Total; Irrelevant; Irreflexive; Asymmetric; Trans;
+  Trichotomous; tri≈; tri<; tri>; _Respects₂_)
+open import Relation.Binary.PropositionalEquality.Core using (_≡_; refl; sym
+  ; cong; cong₂; subst; trans; _≢_)
```

Comment:
```suggestion
open import Relation.Binary.PropositionalEquality.Core
   using (_≡_; refl; sym
          ; cong; cong₂; subst; trans; _≢_)
```
(or at least, with the correct alignment; I'm not seeing it correctly on GitHub)

Reply:
Not sure how/why I suggested this; a single line, although long, seems better?